drivers/node/node.go

```diff
@@ -218,6 +221,9 @@ type Driver interface {
 	// delayInMilliseconds => 1 to 1000
 	InjectNetworkError(nodes []Node, errorInjectionType string, operationType string, dropPercentage int, delayInMilliseconds int) error
 
+	// InjectNetworkError by dropping packets or introdiucing delay in packet tramission and reboot nodes during fallback
```

Comment:
nit: doc string should start with the function name `InjectNetworkErrorWithRebootFallback`